src/atomate2/vasp/jobs/lobster.py

```diff
@@ -151,6 +159,8 @@ def update_user_incar_settings_maker(
         Structure object.
     prev_vasp_dir : Path or str
         Path or string to vasp files.
+    append_name: str or None
```

Comment:
I guess you don't need append_name here since that was for the preconvergence stuff

Reply:
Yes, you are right. I need to clean up.